unit_scaling/_modules.py

```diff
@@ -91,13 +128,17 @@ def __init__(
         self,
         in_features: int,
         out_features: int,
-        bias: bool = True,
+        bias: bool = False,
```

Comment:
why do you deviate from base pytorch here?

Reply:
As discussed, I guess the deviation seems the lesser of two evils.